src/lib/surveyQuestion.ts

```diff
@@ -0,0 +1,19 @@
+export const questions = [
+  { id: 1, text: '다른 팀원의 의견을 경청하고 이해하려고 노력했나요?' },
```

Comment:
```
// surveyModels에 정의
const QUESTION_TYPE = { Radio:'Radio', Check: 'Check', Text: 'Text'} as const;

```

```suggestion
  {  type: QUESTION_TYPE.Radio, id: 1, text: '다른 팀원의 의견을 경청하고 이해하려고 노력했나요?' },
```

나중에 여유 되시면 id 번호로 라디오/체크/텍스트 문항 구분하는 것 보다 타입 하나 추가해서 구분하는게 좋을 것 같습니다!

Reply:
✅ [771a9c5](https://github.com/PRism-Works/PRism-FE/pull/92/commits/771a9c50c4ba39cf604b7968c768a0a029d3e672)